src/service-broker.js

```diff
@@ -186,6 +186,9 @@ class ServiceBroker {
 			// Broker started flag
 			this.started = false;
 
+			/** @type {Boolean} Broker is starting inital services flag*/
+			this.servicesStarted = false;
```

Comment:
The variable name is not represent the exact state because it means services is **starting** right now, so would be better the `servicesStarting` name. But after that, it should set to false when all services started.

Reply:
updated.

Reply:
It's still missing.